packages/peregrine/lib/talons/SearchPage/useSearchPage.js

```diff
@@ -24,17 +24,14 @@ export const useSearchPage = props => {
         }
     } = props;
 
-    const [sort, setSort] = useState({
+    const sortProps = useSort({
```

Comment:
Seems fine to be explicit here, but this is the same as the default in the custom hook now. I would consider removing.